Rakefile

```diff
@@ -262,12 +262,13 @@ def install_prezto
   run %{ ln -nfs "$HOME/.yadr/zsh/prezto" "${ZDOTDIR:-$HOME}/.zprezto" }
 
   # The prezto runcoms are only going to be installed if zprezto has never been installed
-  install_files(Dir.glob('zsh/zshrc'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zlogin'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zlogout'), :symlink)
-  install_files(Dir.glob('zsh/prezto-override/zpreztorc'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zprofile'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zshenv'), :symlink)
+  # Install prezto-override files
+  install_files(Dir.glob('zsh/prezto-override/zpreztorc'), :symlink)
+  install_files(Dir.glob('zsh/prezto-override/zshrc'), :symlink)
 
```

Comment:
Here I moved zshrc file to the prezto-override folder, now it's not necessary to edit zshrc after symlink.